docs/en/observability/apm/view-and-analyze/filter-and-search/cross-cluster-search.asciidoc

```diff
@@ -44,7 +44,25 @@ You can also specify certain clusters to display data from, for example,
 
 There are two ways to edit the default {data-source}:
 
-* In the APM UI -- Navigate to *APM* > *Settings* > *Indices*, and change all `xpack.apm.indices.*` values to
+* In the APM UI -- Navigate to *APM* → *Settings* → *Indices*, and change all `xpack.apm.indices.*` values to
 include remote clusters.
 * In `kibana.yml` -- Update the {kibana-ref}/apm-settings-kb.html[`xpack.apm.indices.*`] configuration values to
 include remote clusters.
+
+// I'm not sure if this is better presented as a troubleshooting topic (i.e. reactive)
+// or a best practice (i.e. preventative). For now I've added it as a tip, but it could
+// be more integrated into the content on this page.
+[TIP]
+.Exclude data tiers from search
+====
+In a cross-cluster search (CCS) environment, it's possible for different clusters to serve different data tiers in responses.
+If one of the requested clusters responds slowly, it can cause a timeout at the proxy after 320 and 502 responses presented
+as failure toast messages in the UI with no data loaded.
```

Comment:
In theory, the problem is not exclusive to CCS envs, but could be potentialized in CSS envs. Searches in the frozen tiers are CPU-intensive, which could lead to slowness and timeouts.

Reply:
If the problem is not exclusive to CCS envs, should this be documented somewhere else? Or should the language here be updated (even though this page is about CCS specifically)?

Reply:
It might be interesting to doc this if there is a doc for the advanced settings, mentioning this more broadly. This section for CCS LGTM.